package.json

```diff
@@ -172,15 +172,15 @@
                 "rust-client.rustupPath": {
                     "type": "string",
                     "default": "rustup",
-                    "description": "Path to rustup executable"
+                    "description": "Path to rustup executable. Ignored if rustup is disabled."
                 },
                 "rust-client.rlsPath": {
                     "type": [
                         "string",
                         "null"
                     ],
                     "default": null,
-                    "description": "Path to rls executable (only required for rls developers)"
+                    "description": "Override RLS path. Only required for RLS developers. If you set this and use rustup, you should also set `rust-client.channel` to ensure your RLS sees the right libraries."
```

Comment:
Is it worth mentioning that if using this for a non-rustup RLS, then you need to separately disable rustup